apps/meteor/app/lib/server/functions/deleteUser.ts

```diff
@@ -61,14 +63,27 @@ export async function deleteUser(userId: string, confirmRelinquish = false): Pro
 
 		Subscriptions.removeByUserId(userId); // Remove user subscriptions
 
+		if (hasRole(userId, 'livechat-agent')) {
```

Comment:
Although there isn't anything wrong with this logic, I feel this is not required at all and we can just call the database directly here without this check & it will not delete anything for users with no `livechat-agent` role. Calling this method is just adding another database call which I don't feel is required here, so this could be a minor optimization here.

Reply:
Well, this also prevents from making a unnecessary call on deletion when the is user is not an agent.